index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="/main.css" type="text/css"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> 
+    <title>Alef's Portfolio</title>
+</head>
+<body>
+
+    <header>
+        <a href="">
+            <img src="./assets/svg/Logo-Placeholder.svg" alt="site logo">
+        </a>
```

Comment:
The logo is currently broken and not showing properly check: [Live demo](https://rawcdn.githack.com/alef-garrido/PortfolioAlef/1073795fdbe580c825b24e0b0ed07572125f00e4/index.html#)